src/config/supported_distros.py

```diff
@@ -37,5 +37,13 @@
 },
 'Rocky Linux': {
 	'RockyLinux 9': 'RockyLinux_9_List.json'
+},
+'IBM Z Validated': {
+    'IBM Z Validated RHEL 8': 'IBM_Validated_OSS_List_RHEL_8.json',
+	'IBM Z Validated RHEL 9': 'IBM_Validated_OSS_List_RHEL_9.json',
+	'IBM Z Validated SLES 12': 'IBM_Validated_OSS_List_SLES_12.json',
+	'IBM Z Validated SLES 15': 'IBM_Validated_OSS_List_SLES_15.json',
+	'IBM Z Validated Ubuntu 20': 'IBM_Validated_OSS_List_Ubuntu_2004.json',
+    'IBM Z Validated Ubuntu 22': 'IBM_Validated_OSS_List_Ubuntu_2204.json'
```

Comment:
These should be Ubuntu 20.04 and 22.04 (the .04 is important, since there were also Ubuntu 20.10 and 22.10 releases).

Reply:
And very minor, but some of the above are spaces and some are tabs, causing the file to be a bit out of whack when tabset is not 4. We've tended to use tabs, so let's stick with that.